src/webapp/components/uploader/js/Uploader.js

```diff
@@ -330,36 +330,36 @@ var fluid_1_4 = fluid_1_4 || {};
             updateStateAfterCompletion(that);
         });
     };
-    
+
     var setupUploader = function (that) {
         that.demo = fluid.typeTag(that.options.demo ? "fluid.uploader.demo" : "fluid.uploader.live");
-        
+
         fluid.initDependents(that);
 
         // Upload button should not be enabled until there are files to upload
         disableElement(that, that.locate("uploadButton"));
         bindDOMEvents(that);
         bindEvents(that);
-        
+
         updateQueueSummaryText(that);
         that.statusUpdater();
         renderFileUploadLimit(that);
-        
-        // Uploader uses application-style keyboard conventions, so give it a suitable role.
-        that.container.attr("role", "application");
+
+        // Uploader file queue uses application-style keyboard conventions, so give it a suitable role.
+        that.locate("fileQueue").attr("role", "application");
```

Comment:
This should actually be done in the fileQueueView. Probably in the [preInit](https://github.com/fluid-project/infusion/blob/infusion-1.4.x/src/webapp/components/uploader/js/FileQueueView.js#L313) or [finalInit](https://github.com/fluid-project/infusion/blob/infusion-1.4.x/src/webapp/components/uploader/js/FileQueueView.js#L301) functions. There you'd do basically the same things as the old code was here `that.container.attr("role", "application");` 

I believe this will also fix the issue you had with the tests.